ontopy/ontology.py

```diff
@@ -395,6 +395,13 @@ def get_by_label_all(self, label, label_annotations=None, prefix=None):
         The current implementation also supports "*" as a wildcard
         matching any number of characters. This may change in the future.
         """
+        if self._label_annotations is None:
+            for iri in DEFAULT_LABEL_ANNOTATIONS:
+                try:
+                    self.add_label_annotation(iri)
+                except ValueError:
+                    pass
```

Comment:
Aren't these lines also repeated in get_by_label()? What about factor them out into a help function?

Reply:
I have moved the lines out, but there is an issue here that should be adressed in a different PR